django_project/frontend/src/pages/Dashboard/MapLibre/Layers/ContextLayers/index.jsx

```diff
@@ -279,13 +279,18 @@ export function ContextLayer({ contextLayerData, map, contextLayerOrder }) {
   const id = ID + '-' + contextLayerData.id
 
   /** CONTEXT LAYER CHANGED */
-  useEffect(() => {
-    if (map && contextLayersShow) {
-      contextLayerRendering(id, contextLayerData, contextLayer, map, contextLayerOrder)
-    } else {
-      removeLayers(map, id)
-    }
-  }, [map, contextLayer, contextLayersShow]);
+  if (map && contextLayersShow) {
+    contextLayerRendering(id, contextLayerData, contextLayer, map, contextLayerOrder)
+  } else {
+    removeLayers(map, id)
+  }
+  // useEffect(() => {
+  //   if (map && contextLayersShow) {
+  //     contextLayerRendering(id, contextLayerData, contextLayer, map, contextLayerOrder)
+  //   } else {
+  //     removeLayers(map, id)
+  //   }
+  // }, [map, contextLayer, contextLayersShow]);
```

Comment:
We also needed to disable this useEffect, I don't know why, but the hook didn´t recongnize the changes in contextLayer.

Reply:
Hi @fpsampayo 
Could you keep this? It prevent the context layer blinking because some of context layer, example: ARGIS, which needs the fetch the data.

Maybe the object are some on context layer and it was just update an object in the object.
Maybe could you try to deep copy the context layer when updating?

Reply:
I really don't understand why the useEffect didn't detect the changes, but i checked and ARCGIS layers didn't blink. Please, try the PR to check with other data.